crates/y-sweet-worker/wrangler.toml

```diff
@@ -47,17 +47,3 @@ bindings = [
 binding = "Y_SWEET_DATA"
 bucket_name = "y-sweet-data-staging"
 preview_bucket_name = "y-sweet-data-dev"
-
-[env.auth-test]
-vars.WORKERS_RS_VERSION = "0.0.11"
```

Comment:
this variable not needed anymore?

Reply:
the entire `auth-test` environment is no longer needed. Instead we just use the test env and pass an additional variable.

Reply:
Yea but the version variable is not defined in the env vars

Reply:
It's defined on line 39 for `[env.test]`
<img width="337" alt="image" src="https://github.com/drifting-in-space/y-sweet/assets/46173/1924fe11-a81c-4807-94fd-1b1261c7cb5e">

The code that invoked `[env.auth-test]` no longer exists

<img width="310" alt="image" src="https://github.com/drifting-in-space/y-sweet/assets/46173/c697df1b-2a35-47bc-a59a-3fccb53b00ea">